src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -14,6 +14,7 @@
 import time as _time
 import six as _six
 
+import turicreate as _turicreate
```

Comment:
Why is this needed?

Reply:
I am calling _turicreate.extensions in create function.

Reply:
Could we just import it in the `create` function then?

Reply:
I will change it